CONTRIBUTING.md

```diff
@@ -60,18 +60,17 @@ This will host an interactive editor at
 [http://localhost:7936/](http://localhost:7936/) to play around with and test
 changes.
 
-#### Jasmine tests
+#### Jest tests
 
-The JavaScript parser and some of the tree
-builder is tested with Jasmine. These tests can be run either using node with
-`make test`, or in the browser by visiting
-[http://localhost:7936/test/test.html](http://localhost:7936/test/test.html).
```

Comment:
I'm using jest in my js tex parser, and I've played around with the node debugging a bit. Lemme know if I can help :)